bin/development-server.js

```diff
@@ -4,14 +4,27 @@
 require("babel-register");
 
 const path = require("path");
+const fs = require("fs");
 const webpack = require("webpack");
 const express = require("express");
 const webpackDevMiddleware = require("webpack-dev-middleware");
 const webpackHotMiddleware = require("webpack-hot-middleware");
 const http = require("http");
 
+// Improve the first run experience: copy the local sample config to a
+// local file so webpack can find it. We don't have to do this, but
+// otherwise webpack prints a warning that scares people.
+const localConfigPath = path.join(__dirname, "../config/local.json")
+if(!fs.existsSync(localConfigPath)) {
+  const defaultConfig = fs.readFileSync(
+    path.join(__dirname, "../config/local.sample.json")
+  );
+  fs.writeFileSync(localConfigPath, defaultConfig);
+}
```

Comment:
I thought of a few different ways to handle config, but couldn't come up with a great solution quickly that works both in client and server. In Node, we could simply check if the local config file exists and if not, don't load it in. But we can't do that in a browser.

We should talk through config soon because I think client config is very different from our server config. The client should ideally take preferences from the server, because in the Firefox toolbox we'll want to give the client preferences from the toolbox.